docs/compatibility.md

```diff
@@ -323,7 +323,7 @@ try to call out which releases we offer different results for. JSON parsing is e
 except for date and timestamp types where we still have work to complete. If you wish to disable
 JSON Scan you can set `spark.rapids.sql.format.json.enabled` or
 `spark.rapids.sql.format.json.read.enabled` to false. To disable `from_json` you can set 
-`spark.rapids.sql.expression.StructsToJson` to false.
+`spark.rapids.sql.expression.JsonToStructs` to false.
```

Comment:
Thanks for the fix. Generally I prefer to have someone post a change request instead of pushing to my PR directly. That is so that I can keep my internal development branch in sync with what is in the PR.